soc/board_specific_workflows/test_ice40up5k.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+# Copyright 2021 The CFU-Playground Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import ice40up5k
+import unittest
+import workflow_args
+from litex.soc.integration import builder
+from unittest import mock
+
+
+class Ice40UP5KWorkflowTest(unittest.TestCase):
+    """TestCase for the Ice40UP5KWorkflow."""
+    def setUp(self):
+        self.args = workflow_args.parse_workflow_args(
+            ['--cpu_variant', 'not fomu + cfu...'])
```

Comment:
This seems specific to the test_warning test(). 

I suggest:

self.args = workflow_args.parse_workflow_args([])

Then add the not-fomu cpu_variant arg directly into the dictionary in the test_warning case.

Reply:
You're correct here, I've gone ahead and moved the cpu_variant into only the test_warning case.